package motor_params_list.msgs;

import "jointmotor_params.proto";

message MotorParamsList {
    repeated joint_motor_params.msgs.JointMotorParams joint_motor_params = 1;
}